beer-schema.sql

```diff
@@ -1,132 +1,127 @@
-CREATE DATABASE IF NOT EXISTS BeerApplication;
-USE BeerApplication;
+-- PM2 Table Creation
+CREATE SCHEMA IF NOT EXISTS BeersApp;
+USE BeersApp;
 
-DROP TABLE IF EXISTS BeerComment;
-DROP TABLE IF EXISTS BeerReview;
 DROP TABLE IF EXISTS ViewHistory;
-DROP TABLE IF EXISTS Beer;
-DROP TABLE IF EXISTS Brewer;
-DROP TABLE IF EXISTS Food;
-DROP TABLE IF EXISTS BeerStyle;
-DROP TABLE IF EXISTS User;
-DROP TABLE IF EXISTS Administrator;
-DROP TABLE IF EXISTS Person;
+DROP TABLE IF EXISTS Persons;
+DROP TABLE IF EXISTS Administrators;
+DROP TABLE IF EXISTS Users;
+DROP TABLE IF EXISTS BeerReviews;
+DROP TABLE IF EXISTS BeerComments;
+DROP TABLE IF EXISTS Brewers;
+DROP TABLE IF EXISTS BeerStyles;
+DROP TABLE IF EXISTS Beers;
+DROP TABLE IF EXISTS Foods;
```

Comment:
This drop order works? Does it not throw an error when you try dropping Persons before Administrators since Administrators references Persons?